root/static/styles/layout.less

```diff
@@ -711,12 +711,12 @@ span.blank-icon {
 }
 
 span.video {
-    background-image: data-uri('../images/icons/video.png');
+    /* based on entity icon styles in entity-icons.less */
+    padding-left: 18px;
```

Comment:
```suggestion
    padding-left: 14px;
    margin-right: 4px;
```

seems to center this in a much nicer way:

<img width="480" height="270" alt="image" src="https://github.com/user-attachments/assets/f2aa7253-a92d-4b66-9ec0-0d627afa8a07" />